src/layouts/UseCases.tsx

```diff
@@ -1,3 +1,448 @@
-// export const useCasesComponents = {}
+import {
+  Badge,
+  Box,
+  calc,
+  chakra,
+  Flex,
+  Heading,
+  Hide,
+  Icon,
+  ListItem,
+  Show,
+  Text,
+  type BoxProps,
+  type FlexProps,
+  type HeadingProps,
+  UnorderedList,
+  useToken,
+} from "@chakra-ui/react"
+import { type ComponentProps } from "react"
+import { MdExpandMore } from "react-icons/md"
 
-export const UseCasesLayout = () => null
+import BannerNotification from "@/components/BannerNotification"
+import ButtonDropdown, {
+  List as ButtonDropdownList,
+  type IProps as ButtonDropdownProps,
+} from "@/components/ButtonDropdown"
+import { Image } from "@/components/Image"
+import { mdxTableComponents } from "@/components/Table"
+import { ButtonLink } from "@/components/Buttons"
+import Card from "@/components/Card"
+import DocLink from "@/components/DocLink"
+import Emoji from "@/components/Emoji"
+import ExpandableCard from "@/components/ExpandableCard"
+import FeedbackCard from "@/components/FeedbackCard"
+import GlossaryTooltip from "@/components/Glossary/GlossaryTooltip"
+import InfoBanner from "@/components/InfoBanner"
+import { BaseLink } from "@/components/Link"
+import MarkdownImage from "@/components/MarkdownImage"
+import MdLink from "@/components/MdLink"
+import MeetupList from "@/components/MeetupList"
+import QuizWidget from "@/components/Quiz/QuizWidget"
+import RandomAppList from "@/components/RandomAppList"
+import SectionNav from "@/components/SectionNav"
+import TableOfContents from "@/components/TableOfContents"
+import UpgradeStatus from "@/components/UpgradeStatus"
+import UpgradeTableOfContents from "@/components/UpgradeTableOfContents"
+import YouTube from "@/components/YouTube"
+// TODO: Import once intl implemented?
+// import PageMetadata from "@/components/PageMetadata"
+
+import { getSummaryPoints } from "@/lib/utils/getSummaryPoints"
+import { isLangRightToLeft } from "@/lib/utils/translations"
+import type { ChildOnlyProp, Lang } from "@/lib/types"
+
+const commonHeadingProps: HeadingProps = {
+  fontWeight: 700,
+  lineHeight: 1.4,
+}
+
+const Heading1 = (props: HeadingProps) => (
+  <Heading as="h1" {...commonHeadingProps} fontSize="2.5rem" {...props} />
```

Comment:
We should keep using the `OldHeading` or set a `mb` to keep the spacing consistent.
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/10e4d568-552b-4504-8877-f914e201c763)

Reply:
To prevent scope creep, will revert to use OldHeading (hah sorry, it's marked as deprecated so it gets crossed out in VSCode... hard to resist)